docs/pom.xml

```diff
@@ -24,12 +24,8 @@
 
 		<!--maven-resources-plugin:3.2.0:copy-resources fails: https://issues.apache.org/jira/browse/MSHARED-966-->
 		<maven-resources-plugin.version>3.3.1</maven-resources-plugin.version>
-		<!-- override version inherited from spring-cloud-build parent to one that is available on Maven Central -->
-		<spring-asciidoctor-backends.version>0.0.4</spring-asciidoctor-backends.version>
```

Comment:
This is not necessary as the asciidoctor backend version (0.0.5) inherited from spring-cloud-build is available on Maven Central.